dashboard/app/mailers/follower_mailer.rb

```diff
@@ -1,4 +1,5 @@
 class FollowerMailer < ActionMailer::Base
+  include ActionMailerMetrics
```

Comment:
I would recommend creating [dashboard/app/mailers/application_mailer.rb](https://guides.rubyonrails.org/action_mailer_basics.html#create-the-mailer) and inheriting all the mailers from it:

```ruby
class ApplicationMailer < ActionMailer::Base
  include ActionMailerMetrics
end
```

```suggestion
class FollowerMailer < ApplicationMailer
```

Reply:
good idea, done